app/controllers/episodes_controller.rb

```diff
@@ -12,6 +12,12 @@ def edit; end
 
   def new; end
 
+  def download
+    @episode = Episode.find(params[:episode_id])
```

Comment:
You have @episode form `set_episode`

Reply:
It work as you say with `on: :member` in routes